components/notification/overview.md

```diff
@@ -201,6 +187,14 @@ You can find more information on opening, closing and hiding the Notification in
 
 >tip @[template](/_contentTemplates/notification/templates.md#one-instance-per-app-link)
 
+## Templates
+
+You can control the rendering of the Notification by using Templates. [See the Templates article for more information...]({%slug notification-templates%})
+
+## Styling and Appearance
```

Comment:
Feature sections should precede the Parameters sections. This one about Styling duplicates one above (Success, Info, Warning, Error Notifications), which in turn is too detailed and duplicates content in the Appearance article. Consider refactoring and optimization.